CRITICAL - 2020-04-19 00:12:45 --> Namespace declaration statement has to be the very first statement or after any declare call in the script
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-04-19 00:13:22 --> CodeIgniter\Router\Router::main(): Failed opening required 'application/libraries/CryptoLib/src/CryptoLib.php' (include_path='.:')
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-04-19 00:25:10 --> CodeIgniter\Router\Router::main(): Failed opening required 'application/libraries/CryptoLib/src/CryptoLib.php' (include_path='.:')
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-04-19 00:27:18 --> Unknown column 'a.name' in 'field list'
#0 /Library/WebServer/Documents/digitaltax/system/Database/mysqli/Connection.php(330): mysqli->query('SELECT  a.id,\n ...')
#1 /Library/WebServer/Documents/digitaltax/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT  a.id,\n ...')
#2 /Library/WebServer/Documents/digitaltax/system/Database/BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT  a.id,\n ...')
#3 /Library/WebServer/Documents/digitaltax/app/Models/Authentication/AuthenticationModel.php(69): CodeIgniter\Database\BaseConnection->query('SELECT  a.id,\n ...')
#4 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(22): App\Models\Authentication\AuthenticationModel->authenticate('confetti.tech15...', 123456, 1, false)
#5 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#6 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#9 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#10 {main}
CRITICAL - 2020-04-19 00:34:45 --> Unknown column 'a.tcoinBalance' in 'field list'
#0 /Library/WebServer/Documents/digitaltax/system/Database/mysqli/Connection.php(330): mysqli->query('SELECT  a.id,\n ...')
#1 /Library/WebServer/Documents/digitaltax/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT  a.id,\n ...')
#2 /Library/WebServer/Documents/digitaltax/system/Database/BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT  a.id,\n ...')
#3 /Library/WebServer/Documents/digitaltax/app/Models/Authentication/AuthenticationModel.php(68): CodeIgniter\Database\BaseConnection->query('SELECT  a.id,\n ...')
#4 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(22): App\Models\Authentication\AuthenticationModel->authenticate('confetti.tech15...', 123456, 1, false)
#5 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#6 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#9 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#10 {main}
CRITICAL - 2020-04-19 00:35:55 --> You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near '.updated_at,
                    a.deleted_at,
                    b.`code` as u' at line 14
#0 /Library/WebServer/Documents/digitaltax/system/Database/mysqli/Connection.php(330): mysqli->query('SELECT  a.id,\n ...')
#1 /Library/WebServer/Documents/digitaltax/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT  a.id,\n ...')
#2 /Library/WebServer/Documents/digitaltax/system/Database/BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT  a.id,\n ...')
#3 /Library/WebServer/Documents/digitaltax/app/Models/Authentication/AuthenticationModel.php(66): CodeIgniter\Database\BaseConnection->query('SELECT  a.id,\n ...')
#4 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(22): App\Models\Authentication\AuthenticationModel->authenticate('confetti.tech15...', 123456, 1, false)
#5 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#6 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#7 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#9 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#10 {main}
CRITICAL - 2020-04-19 00:37:22 --> Unknown column 'created' in 'where clause'
#0 /Library/WebServer/Documents/digitaltax/system/Database/mysqli/Connection.php(330): mysqli->query('SELECT * FROM `...')
#1 /Library/WebServer/Documents/digitaltax/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT * FROM `...')
#2 /Library/WebServer/Documents/digitaltax/system/Database/BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT * FROM `...')
#3 /Library/WebServer/Documents/digitaltax/app/Models/Authentication/AuthenticationModel.php(82): CodeIgniter\Database\BaseConnection->query('SELECT * FROM `...')
#4 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(283): App\Models\Authentication\AuthenticationModel->checkIfSessionAlreadyActiveByUserId('1', 1)
#5 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(33): App\Controllers\Auth->getAccessTokenForLogin(Array, 1)
#6 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#7 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#8 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#10 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#11 {main}
CRITICAL - 2020-04-19 00:38:49 --> Class 'App\Controllers\Encryption' not found
#0 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(286): App\Controllers\Auth->generateAuthToken(Array)
#1 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(33): App\Controllers\Auth->getAccessTokenForLogin(Array, 1)
#2 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#3 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#4 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#6 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#7 {main}
CRITICAL - 2020-04-19 00:39:55 --> Class 'App\Controllers\Encryption' not found
#0 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(286): App\Controllers\Auth->generateAuthToken(Array)
#1 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(33): App\Controllers\Auth->getAccessTokenForLogin(Array, 1)
#2 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#3 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#4 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#6 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#7 {main}
CRITICAL - 2020-04-19 00:40:30 --> Call to undefined method CodeIgniter\Encryption\Handlers\OpenSSLHandler::createKey()
#0 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(286): App\Controllers\Auth->generateAuthToken(Array)
#1 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(33): App\Controllers\Auth->getAccessTokenForLogin(Array, 1)
#2 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#3 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#4 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#6 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#7 {main}
CRITICAL - 2020-04-19 00:41:51 --> Class 'Config\Services\Encryption' not found
#0 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(286): App\Controllers\Auth->generateAuthToken(Array)
#1 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(33): App\Controllers\Auth->getAccessTokenForLogin(Array, 1)
#2 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#3 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#4 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#6 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#7 {main}
CRITICAL - 2020-04-19 00:44:40 --> Use of undefined constant app - assumed 'app' (this will throw an Error in a future version of PHP)
#0 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(296): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Use of undefine...', '/Library/WebSer...', 296, Array)
#1 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(286): App\Controllers\Auth->generateAuthToken(Array)
#2 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(33): App\Controllers\Auth->getAccessTokenForLogin(Array, 1)
#3 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#4 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#5 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#7 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#8 {main}
CRITICAL - 2020-04-19 00:46:01 --> Call to undefined method Config\Encryption::createKey()
#0 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(286): App\Controllers\Auth->generateAuthToken(Array)
#1 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(33): App\Controllers\Auth->getAccessTokenForLogin(Array, 1)
#2 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#3 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#4 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#6 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#7 {main}
CRITICAL - 2020-04-19 00:51:41 --> Unknown column 'active' in 'field list'
#0 /Library/WebServer/Documents/digitaltax/system/Database/mysqli/Connection.php(330): mysqli->query('INSERT INTO `us...')
#1 /Library/WebServer/Documents/digitaltax/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Library/WebServer/Documents/digitaltax/system/Database/BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Library/WebServer/Documents/digitaltax/app/Models/Authentication/AuthenticationModel.php(147): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...')
#4 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(108): App\Models\Authentication\AuthenticationModel->saveAccessToken('1', '\xB2\x18\x0F\xCF\xEF\xC5/Em$\x92>\x06\xAC\xAE...', '1', 0)
#5 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(287): App\Controllers\Auth->saveAccessToken('1', '\xB2\x18\x0F\xCF\xEF\xC5/Em$\x92>\x06\xAC\xAE...', '1', 0)
#6 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(33): App\Controllers\Auth->getAccessTokenForLogin(Array, 1)
#7 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#8 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#11 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#12 {main}
CRITICAL - 2020-04-19 00:51:41 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "{0}". in /Library/WebServer/Documents/digitaltax/system/Format/Exceptions/FormatException.php:9
Stack trace:
#0 /Library/WebServer/Documents/digitaltax/system/Format/JSONFormatter.php(67): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 /Library/WebServer/Documents/digitaltax/system/API/ResponseTrait.php(387): CodeIgniter\Format\JSONFormatter->format(Array)
#2 /Library/WebServer/Documents/digitaltax/system/API/ResponseTrait.php(125): CodeIgniter\Debug\Exceptions->format(Array)
#3 /Library/WebServer/Documents/digitaltax/system/Debug/Exceptions.php(165): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(mysqli_sql_exception))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-04-19 00:58:35 --> Column count doesn't match value count at row 1
#0 /Library/WebServer/Documents/digitaltax/system/Database/mysqli/Connection.php(330): mysqli->query('INSERT INTO `us...')
#1 /Library/WebServer/Documents/digitaltax/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Library/WebServer/Documents/digitaltax/system/Database/BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Library/WebServer/Documents/digitaltax/app/Models/Authentication/AuthenticationModel.php(145): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...')
#4 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(108): App\Models\Authentication\AuthenticationModel->saveAccessToken('1', '3\x04\xAB2q\xA8\xFF\x9EqA\x85 \x8E\x1E^...', '1', 0)
#5 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(289): App\Controllers\Auth->saveAccessToken('1', '3\x04\xAB2q\xA8\xFF\x9EqA\x85 \x8E\x1E^...', '1', 0)
#6 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(33): App\Controllers\Auth->getAccessTokenForLogin(Array, 1)
#7 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#8 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#11 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#12 {main}
CRITICAL - 2020-04-19 00:58:35 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "{0}". in /Library/WebServer/Documents/digitaltax/system/Format/Exceptions/FormatException.php:9
Stack trace:
#0 /Library/WebServer/Documents/digitaltax/system/Format/JSONFormatter.php(67): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 /Library/WebServer/Documents/digitaltax/system/API/ResponseTrait.php(387): CodeIgniter\Format\JSONFormatter->format(Array)
#2 /Library/WebServer/Documents/digitaltax/system/API/ResponseTrait.php(125): CodeIgniter\Debug\Exceptions->format(Array)
#3 /Library/WebServer/Documents/digitaltax/system/Debug/Exceptions.php(165): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(mysqli_sql_exception))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-04-19 01:00:09 --> Column count doesn't match value count at row 1
#0 /Library/WebServer/Documents/digitaltax/system/Database/mysqli/Connection.php(330): mysqli->query('INSERT INTO `us...')
#1 /Library/WebServer/Documents/digitaltax/system/Database/BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 /Library/WebServer/Documents/digitaltax/system/Database/BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 /Library/WebServer/Documents/digitaltax/app/Models/Authentication/AuthenticationModel.php(145): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...')
#4 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(108): App\Models\Authentication\AuthenticationModel->saveAccessToken('1', '\x9A@\xC0:j\x97\xD7\x89\xA2\xC9\xF4y\xCBR\xEA...', '1', 0)
#5 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(289): App\Controllers\Auth->saveAccessToken('1', '\x9A@\xC0:j\x97\xD7\x89\xA2\xC9\xF4y\xCBR\xEA...', '1', 0)
#6 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(33): App\Controllers\Auth->getAccessTokenForLogin(Array, 1)
#7 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#8 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#9 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#11 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#12 {main}
CRITICAL - 2020-04-19 01:00:09 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "{0}". in /Library/WebServer/Documents/digitaltax/system/Format/Exceptions/FormatException.php:9
Stack trace:
#0 /Library/WebServer/Documents/digitaltax/system/Format/JSONFormatter.php(67): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 /Library/WebServer/Documents/digitaltax/system/API/ResponseTrait.php(387): CodeIgniter\Format\JSONFormatter->format(Array)
#2 /Library/WebServer/Documents/digitaltax/system/API/ResponseTrait.php(125): CodeIgniter\Debug\Exceptions->format(Array)
#3 /Library/WebServer/Documents/digitaltax/system/Debug/Exceptions.php(165): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(mysqli_sql_exception))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-04-19 01:02:15 --> Undefined variable: sql
#0 /Library/WebServer/Documents/digitaltax/app/Models/Authentication/AuthenticationModel.php(147): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', '/Library/WebSer...', 147, Array)
#1 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(108): App\Models\Authentication\AuthenticationModel->saveAccessToken('1', '\xB4\xFED,\xA0v\xC0]{*-\xD7\x9AQ\xEA...', '1', 0)
#2 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(289): App\Controllers\Auth->saveAccessToken('1', '\xB4\xFED,\xA0v\xC0]{*-\xD7\x9AQ\xEA...', '1', 0)
#3 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(33): App\Controllers\Auth->getAccessTokenForLogin(Array, 1)
#4 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#5 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#8 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#9 {main}
CRITICAL - 2020-04-19 01:02:15 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "{0}". in /Library/WebServer/Documents/digitaltax/system/Format/Exceptions/FormatException.php:9
Stack trace:
#0 /Library/WebServer/Documents/digitaltax/system/Format/JSONFormatter.php(67): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 /Library/WebServer/Documents/digitaltax/system/API/ResponseTrait.php(387): CodeIgniter\Format\JSONFormatter->format(Array)
#2 /Library/WebServer/Documents/digitaltax/system/API/ResponseTrait.php(125): CodeIgniter\Debug\Exceptions->format(Array)
#3 /Library/WebServer/Documents/digitaltax/system/Debug/Exceptions.php(165): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-04-19 01:03:14 --> Undefined variable: sql
#0 /Library/WebServer/Documents/digitaltax/app/Models/Authentication/AuthenticationModel.php(146): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', '/Library/WebSer...', 146, Array)
#1 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(108): App\Models\Authentication\AuthenticationModel->saveAccessToken('1', '\x8A\xD1lz\x02\xB7G\xB1\xC8\xFF\xD6\xC8T6\xB3...', '1', 0)
#2 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(289): App\Controllers\Auth->saveAccessToken('1', '\x8A\xD1lz\x02\xB7G\xB1\xC8\xFF\xD6\xC8T6\xB3...', '1', 0)
#3 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(33): App\Controllers\Auth->getAccessTokenForLogin(Array, 1)
#4 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#5 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#8 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#9 {main}
CRITICAL - 2020-04-19 01:03:14 --> Uncaught CodeIgniter\Format\Exceptions\FormatException: Failed to parse json string, error: "{0}". in /Library/WebServer/Documents/digitaltax/system/Format/Exceptions/FormatException.php:9
Stack trace:
#0 /Library/WebServer/Documents/digitaltax/system/Format/JSONFormatter.php(67): CodeIgniter\Format\Exceptions\FormatException::forInvalidJSON('Malformed UTF-8...')
#1 /Library/WebServer/Documents/digitaltax/system/API/ResponseTrait.php(387): CodeIgniter\Format\JSONFormatter->format(Array)
#2 /Library/WebServer/Documents/digitaltax/system/API/ResponseTrait.php(125): CodeIgniter\Debug\Exceptions->format(Array)
#3 /Library/WebServer/Documents/digitaltax/system/Debug/Exceptions.php(165): CodeIgniter\Debug\Exceptions->respond(Array, 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ErrorException))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-04-19 01:13:12 --> Undefined variable: sql
#0 /Library/WebServer/Documents/digitaltax/app/Models/Authentication/AuthenticationModel.php(147): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', '/Library/WebSer...', 147, Array)
#1 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(108): App\Models\Authentication\AuthenticationModel->saveAccessToken('1', 'adfwregetrthrhr...', '1', 0)
#2 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(291): App\Controllers\Auth->saveAccessToken('1', 'adfwregetrthrhr...', '1', 0)
#3 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(33): App\Controllers\Auth->getAccessTokenForLogin(Array, 1)
#4 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#5 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#6 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#8 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#9 {main}
CRITICAL - 2020-04-19 01:13:36 --> Undefined index: access_token
#0 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(35): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', '/Library/WebSer...', 35, Array)
#1 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#2 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#5 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#6 {main}
CRITICAL - 2020-04-19 01:14:41 --> Undefined index: created
#0 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(37): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', '/Library/WebSer...', 37, Array)
#1 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#2 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#5 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#6 {main}
CRITICAL - 2020-04-19 20:38:06 --> Undefined index: access_token
#0 /Library/WebServer/Documents/digitaltax/app/Controllers/Auth.php(36): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', '/Library/WebSer...', 36, Array)
#1 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(910): App\Controllers\Auth->webSignin()
#2 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Auth))
#3 /Library/WebServer/Documents/digitaltax/system/CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 /Library/WebServer/Documents/digitaltax/public/index.php(45): CodeIgniter\CodeIgniter->run()
#5 /Library/WebServer/Documents/digitaltax/system/Commands/Server/rewrite.php(34): require_once('/Library/WebSer...')
#6 {main}
